modular_skyrat/code/game/objects/items/storage/MREs.dm

```diff
@@ -0,0 +1,98 @@
+/obj/item/storage/mre
+	name = "standard MRE"
+	desc = "A vacuum-sealed bag containing a day's worth of nutrients for a crewmember in strenuous situations. There is no visible expiration date on the package."
+	icon = 'modular_skyrat/icons/obj/MRE.dmi'
+	icon_state = "mre"
+
+/obj/item/storage/mre/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	var/static/list/can_hold = typecacheof(list(
+		/obj/item/reagent_containers/food,
+		/obj/item/reagent_containers/glass/beaker/waterbottle,
+		/obj/item/kitchen/fork,
+		/obj/item/reagent_containers/food/condiment/pack/))
+	STR.can_hold = can_hold
+
+/obj/item/storage/mre/PopulateContents()
+	new /obj/item/reagent_containers/food/snacks/soup/mre(src)
+	switch (rand(1,2))
+		if(1)
+			new /obj/item/reagent_containers/food/snacks/sosjerky/healthy/mre(src)
+		if(2)
+			new /obj/item/reagent_containers/food/snacks/no_raisin/healthy/mre(src)
+	new /obj/item/reagent_containers/food/snacks/honeybar(src)
+	new /obj/item/reagent_containers/glass/beaker/waterbottle(src)
+	new /obj/item/kitchen/fork(src)
+
+/obj/item/storage/mre/protein
+	name = "protein MRE"
+	icon_state = "meatmre"
+
+/obj/item/storage/mre/protein/PopulateContents()
+	switch (rand(1,5))
+		if(1)
+			new /obj/item/reagent_containers/food/snacks/meatcube(src)
+		if(2)
+			new /obj/item/reagent_containers/food/snacks/bbqribs(src)
+		if(3)
+			new /obj/item/reagent_containers/food/snacks/cubancarp(src)
+		if(4)
+			new /obj/item/reagent_containers/food/snacks/taco/plain(src)
+		if(5)
+			new /obj/item/reagent_containers/food/snacks/burger/plain(src)
+	new /obj/item/reagent_containers/food/snacks/sosjerky/healthy/mre(src)
+	new /obj/item/reagent_containers/food/snacks/honeybar(src)
+	new /obj/item/reagent_containers/glass/beaker/waterbottle(src)
+	new /obj/item/reagent_containers/food/condiment/pack/ketchup(src)
+	new /obj/item/reagent_containers/food/condiment/pack/bbqsauce(src)
+	new /obj/item/kitchen/fork(src)
+
+/obj/item/storage/mre/vegie
+	name = "vegetarian MRE"
+	icon_state = "vegmre"
+
+/obj/item/storage/mre/vegie/PopulateContents()
+	switch (rand(1,5))
+		if(1)
+			new /obj/item/reagent_containers/food/snacks/salad/herbsalad(src)
+		if(2)
+			new /obj/item/reagent_containers/food/snacks/salad/citrusdelight(src)
+		if(3)
+			new /obj/item/reagent_containers/food/snacks/loadedbakedpotato(src)
+		if(4)
+			new /obj/item/reagent_containers/food/snacks/carrotfries(src)
+		if(5)
+			new /obj/item/reagent_containers/food/snacks/salad/boiledrice(src)
```

Comment:
same here

Reply:
tried pick() and it didn't work, I showed you this earlier.